### DISCUSSION TITLE
About the fixed forge version

### DISCUSSION THREAD

**Discussion by lockf** [Sinytra/Connector#42] [Q&A] [answered]
Is there any specific reason for the only forge that can be used is 47.1.3 or is it just for control of compatability ?
Can't the mod be made to use forge 47.1.+ ?

I wanted to use the connector but since i have mods that require a more recent version of forge it doesn't work .

**Comment by Su5eD** [comment, marked as answer]
Usually, we would target the recommended version, which is generally considered stable. However, the situation in 1.20.1 is a little complicated. Forge version `47.1.3` is the last version released by what previously used to be the MinecraftForge team. After NeoForge was created, upstream Forge descended into chaos where untested changes were accepted into the existing recommended release. This led to issues on newer Forge versions and had many modders decide on `47.1.3` as the "last stable version". On the other hand, as you mentioned, there are also mods that explicitly require a later Forge version and are not compatible with `47.1.3`. 

In our case, the Forgified Fabric API contains a mixin that conflicts with new Forge versions and therefore is not compatible with them. Under normal circumstances, I could simply raise the version requirement without having to worry about conflicts. In this situation though, we had to make a choice between either `47.1.3` or latest Forge. I decided to go with the former as it's guaranteed to be stable, and modpacks also often pick this version.

In the end, Connector is still unstable and not ready for production. So don't worry, you're not missing out on anything :) Eventually, the community will move on to the next mc version, and we can hopefully switch to NeoForged as well.

**Reply by lockf** [reply]
Understood

**Comment by yozurukt** [comment]
It appears that there is no version of `Optifine` available that supports Forge `47.1.3`, but for me personally, the latter seems to be the more essential one. QAQ

**Reply by kakashi1464** [reply]
Use embeddium with oculus

**Reply by yozurukt** [reply]
Thank you very much, this seems to be a good solution as opposed to using optifine. (It causes too many compatibility issues)
